textwiser/__init__.py

```diff
@@ -20,7 +20,7 @@
 """
 
 __author__ = "FMR LLC"
-__email__ = "textwiser@fmr.com"
+__email__ = "opensource@fidelity.com"
```

Comment:
@skadio should we also be using this email for mabwiser?

Reply:
what's this email?